README.md

```diff
@@ -298,6 +188,19 @@ resp, err := client.Search(indexUID).Search(meilisearch.SearchRequest{
 }
 ```
 
+## 🤖 Compatibility with MeiliSearch
+
+This package only guarantees the compatibility with the [version v0.15.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.15.0).
+
+## 💡 Learn More
+
+The following sections may interest you:
+
+- **Manipulate documents**: see the [API references](https://docs.meilisearch.com/references/documents.html) or read more about the [document concept](https://docs.meilisearch.com/guides/main_concepts/documents.html).
+- **Search**: see the [API references](https://docs.meilisearch.com/references/search.html) or follow our [guide on search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
+- **Manage the indexes**: see the [API references](https://docs.meilisearch.com/references/indexes.html) or read more about the [index concept](https://docs.meilisearch.com/guides/main_concepts/indexes.html).
+- **Configure the index settings**: see the [API references](https://docs.meilisearch.com/references/settings.html) or follow our [guide on settings parameters](https://docs.meilisearch.com/guides/advanced_guides/settings.html#synonyms).
```

Comment:
```suggestion
- **Manipulate documents**: see the [API references](https://docs.meilisearch.com/references/documents.html) or read more about [documents](https://docs.meilisearch.com/guides/main_concepts/documents.html).
- **Search**: see the [API references](https://docs.meilisearch.com/references/search.html) or follow our [guide on search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
- **Manage the indexes**: see the [API references](https://docs.meilisearch.com/references/indexes.html) or read more about [indexes](https://docs.meilisearch.com/guides/main_concepts/indexes.html).
- **Configure the index settings**: see the [API references](https://docs.meilisearch.com/references/settings.html) or follow our [guide on settings parameters](https://docs.meilisearch.com/guides/advanced_guides/settings.html#synonyms).
```

Reply:
Waiting for @bidoubiwa's opinion! 🙂

Reply:
Just as a note, notice that I removed the `#synonyms` from the last link, I think it should be removed in any case

Reply:
Oh yes! Well spotted!!

Reply:
Both are nice but I prefer eskombro's for readability.